pipelines/skylab/slideseq/test_inputs/Plumbing/slideseq_downsample.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
```

Comment:
Do we want to keep this script here or under pytools dockers? (ultimately as part of warp-tools repo)

Reply:
@khajoue2 I guess my thought is that this script is very specific to the plumbing test inputs, which isn't exactly necessary for the pipeline itself. I think it makes sense to have the script that was used to make the data sit next to the data itself. But if we decide to move it, we just need to be clear where to find it.